src/screens/Exchange/ExchangeConfirm.js

```diff
@@ -145,8 +145,8 @@ const ExchangeConfirmScreen = ({
   const fetchTransactionEstimate = () => {
     const isConvertedTx = isWethConvertedTx(symbol, contractAddress);
 
-    // for WETH converted txs on homestead, we need to provide ETH data or else estimation is always 0$
-    const contractAddressForEstimation = isProdEnv && isConvertedTx
+    // for WETH converted txs on homestead, we need to provide ETH data or else estimation fails
+    const contractAddressForEstimation = getEnv().NETWORK_PROVIDER === 'homestead' && isConvertedTx
```

Comment:
```suggestion
    const contractAddressForEstimation = isProdEnv() && isConvertedTx
```
Consider chaning isProdEnv into function or adding a new function with different name. This check seems to really on implementation detail.